src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -229,15 +229,36 @@ public void run() {
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.TungstenSteel, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Tin.getFluid(144), CustomItemList.Casing_AirFilter_Vent_T3.get(1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.TungstenSteel, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Lead.getFluid(288), CustomItemList.Casing_AirFilter_Vent_T3.get(1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{CustomItemList.Casing_AirFilter_Vent_T3.get(1L), GT_OreDictUnificator.get(OrePrefixes.stickLong, Materials.TungstenSteel, 2L), ItemList.Electric_Motor_IV.get(2L), GT_Utility.getIntegratedCircuit(1)}, Materials.SolderingAlloy.getFluid(72), CustomItemList.Casing_AirFilter_Turbine_T3.get(1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{CustomItemList.Casing_AirFilter_Vent_T3.get(1L), GT_OreDictUnificator.get(OrePrefixes.stickLong, Materials.TungstenSteel, 2L), ItemList.Electric_Motor_IV.get(2L), GT_Utility.getIntegratedCircuit(1)}, Materials.Tin.getFluid(144), CustomItemList.Casing_AirFilter_Turbine_T3.get(1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{CustomItemList.Casing_AirFilter_Vent_T3.get(1L), GT_OreDictUnificator.get(OrePrefixes.stickLong, Materials.TungstenSteel, 2L), ItemList.Electric_Motor_IV.get(2L), GT_Utility.getIntegratedCircuit(1)}, Materials.Lead.getFluid(288), CustomItemList.Casing_AirFilter_Turbine_T3.get(1L), 100, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{ItemList.Hull_IV.get(1L), ItemList.Casing_Turbine3.get(1L), GT_OreDictUnificator.get(OrePrefixes.rotor, Materials.TungstenSteel, 1L), ItemList.Electric_Motor_IV.get(2L), ItemList.Electric_Pump_IV.get(1L), GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.Tungsten, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.SolderingAlloy.getFluid(144), CustomItemList.Machine_Multi_AirFilterT3.get(1L), 200, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{ItemList.Hull_IV.get(1L), ItemList.Casing_Turbine3.get(1L), GT_OreDictUnificator.get(OrePrefixes.rotor, Materials.TungstenSteel, 1L), ItemList.Electric_Motor_IV.get(2L), ItemList.Electric_Pump_IV.get(1L), GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.Tungsten, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Tin.getFluid(288), CustomItemList.Machine_Multi_AirFilterT3.get(1L), 200, 120);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{ItemList.Hull_IV.get(1L), ItemList.Casing_Turbine3.get(1L), GT_OreDictUnificator.get(OrePrefixes.rotor, Materials.TungstenSteel, 1L), ItemList.Electric_Motor_IV.get(2L), ItemList.Electric_Pump_IV.get(1L), GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.Tungsten, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Lead.getFluid(576), CustomItemList.Machine_Multi_AirFilterT3.get(1L), 200, 120);
```

Comment:
If someone could tell me why the fluids aren't added in the recipe in game, because despite adding 3 recipes for each block in the assembler, i only get one with no fluid.

Reply:
you should use getMolten()